editor/src/components/canvas/controls/comment-popup.tsx

```diff
@@ -57,8 +60,11 @@ interface CommentThreadProps {
 
 function CommentThread({ x, y }: CommentThreadProps) {
   const thread = useCanvasCommentThread(x, y)
+
   const createThread = useCreateThread()
 
+  const colorIndex = useMyMultiplayerColorIndex() ?? -1
```

Comment:
it would be better to either default this to `0` or (better?) `null` to be able to pick a good color from the colors list

Reply:
Unfortunately, the comment metadata base type does not allow nullable properties, only string, number and boolean. And I wanted the helper to return the fallback color, and not the first one (which is a hack though....)

Reply:
Can you add a comment here explaining this please, because I can definitely see someone changing this later without realising why the `-1` is a deliberate choice